### ISSUE TITLE
Integrating with liquid-fire

### ISSUE THREAD

**Issue by knownasilya** [nathanhammond/ember-a11y#5]
Do you plan on supporting liquid-fire outlets, i.e. `{{liquid-outlet}}`?

**Comment by nathanhammond** [comment]
@ef4 and I talked about this super-briefly. I'm not sure what the correct solution is for that scenario. We need the same primitives in order to accomplish this task, but it's super-easy to stomp all over each other.

It's that same primitive that made it super easy to implement `{{liquid-modal}}` in liquid-fire even though it has nothing to do with animation. And yet, modals are something really difficult to build accessibly and should probably be addressed inside of ember-a11y. Currently liquid-fire is in the process of removing liquid-modal, I'm likely to add an implementation of modals here.

Regardless, at some point this will be compatible with liquid-fire, but my imagination says that liquid-fire will likely extend some sort of base functionality from ember-a11y as it will have its own separate accessibility concerns to address.

**Comment by knownasilya** [comment]
Makes sense to do it the other way around, using this addon to build off of in liquid-fire.
